setup/context/commands_test.go

```diff
@@ -0,0 +1,69 @@
+package context
```

Comment:
I had a feeling you would say as much about `gomock`. I think @jonathanstallings work here is valuable in learning _why_ we use Gomock, and in that sense, this was not a pointless exercise. As for the value of the tests themselves and the "all or nothing" approach, I'm inclined to agree with you.

Devil's advocate argument re: `MockIO` is that for new developers coming into this project, the need for Gomock is not entirely obvious. I'd like us to keep that in mind when people attempt to add tests in the future (if they even try to add tests!)